AGENTS.md

```
@@ -1,53 +1,52 @@
 
-## Workflow Tasks
+## Architecture
 
-- when summarizing changes only include details that require further action
-- just say "Done." when there is no further action
-- use the github CLI `gh` to create pull requests and work with github
-- Rules for creating pull requests:
-  - keep them short and focused on changes.
-  - never include a test plan
-  - write the summary using the same style rules as commit message
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)